src/components/modals/FlipInputModal.js

```diff
@@ -106,8 +109,9 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
   }
 
   renderExchangeRates = () => {
-    const { primaryInfo, secondaryInfo, fiatPerCrypto, errorMessage, theme } = this.props
+    const { primaryInfo, secondaryInfo, fiatPerCrypto, theme } = this.props
```

Comment:
Instead of removing the "errorMessage" from the destructor, you can use a default:
```
const { 
   primaryInfo,
   secondaryInfo, 
   fiatPerCrypto,
   errorMessage = this.state.errorMessage, 
   theme 
} = this.props
```